test/test-PreviewProvider.js

```diff
@@ -199,6 +199,9 @@ exports.test_process_links = function(assert) {
 exports.test_process_and_insert_links = function(assert) {
   const fakeData = {"url": "http://example.com/1", "title": "Title for example.com/1"};
 
+  const mockStoreActions = [];
+  gPreviewProvider._store = {dispatch: action => mockStoreActions.push(action)};
```

Comment:
This passes locally on my machine, I suspect it's because of some JS behaviour difference with FFox 46